### ISSUE TITLE
Information conflict

### ISSUE THREAD

**Issue by Scaledi** [LemmyNet/lemmy#1004]
### Issue Summary

Lemmy databases seem to have a unresolved conflict. On my portfolio (u/cloak) it says I moderate c/philosophy. Upon visiting the community, I am not a moderator.

### Steps to Reproduce

1. Vist u/Cloak
2. Click on c/philosophy from community’s he moderates 
3. Note how I am not a mod

### Technical details

On iPad os, as well as arch Linux on Firefox

**Comment by Scaledi** [comment]
Found another issue like this. Pls close

**Comment by dessalines** [comment]
Figured this one out. There's a duplicate community here:

```
lemmy=# select id, name from community where name = 'philosophy';
  id   |    name
-------+------------
 15257 | philosophy
 15972 | philosophy
(2 rows)
```

This will be fixed when we deploy #994 , it will delete your philosophy community and preserve the first one made.

**Comment by Scaledi** [comment]
Cool
